packages/main/src/plugin/extension-loader.ts

```diff
@@ -1000,6 +1001,9 @@ export class ExtensionLoader {
       createPod(podOptions: containerDesktopAPI.PodCreateOptions): Promise<{ engineId: string; Id: string }> {
         return containerProviderRegistry.createPod(podOptions);
       },
+      listPods(): Promise<PodInfo[]> {
+        return containerProviderRegistry.listPods();
+      },
```

Comment:
it's probably missing also a test in `extension-loader.spec.ts`


checking that we call listPods on containerProviderRegistry